projects/1.18/assets/macaws-roofs-biomes-o-plenty/macawsroofsbop/lang/zh_cn.json

```
@@ -0,0 +1,187 @@
+{
+	"itemGroup.tab_mrbop": "Macaw的屋顶-BOP附属",
+
+    "block.macawsroofsbop.cherry_attic_roof": "樱桃木天窗屋顶",
+	"block.macawsroofsbop.cherry_planks_attic_roof": "樱桃木木板天窗屋顶",
+	"block.macawsroofsbop.dead_attic_roof": "枯死木天窗屋顶",
+	"block.macawsroofsbop.dead_planks_attic_roof": "枯死木木板天窗屋顶",
+	"block.macawsroofsbop.fir_attic_roof": "杉木天窗屋顶",
+	"block.macawsroofsbop.fir_planks_attic_roof": "杉木木板天窗屋顶",
+	"block.macawsroofsbop.hellbark_attic_roof": "地狱皮天窗屋顶",

```

```suggestion
	"block.macawsroofsbop.hellbark_attic_roof": "地狱皮木天窗屋顶",
```
下同，缺木统一补
`木板xx屋顶`这个是一个完整的木制品名，不用考虑重复